src/main/java/me/amarpandey/controller/ChatController.java

```diff
@@ -43,7 +43,7 @@ public UserResponse getMessage(@Payload UserResponse userResponse) {
 	@MessageMapping("{groupid}/connect")
 	@SendTo("/topic/{groupid}/connect")
 	public UserResponse connect(@DestinationVariable String groupid, @RequestParam String username,
-			SimpMessageHeaderAccessor headerAccessor) {
+								SimpMessageHeaderAccessor headerAccessor) {
```

Comment:
what's the need for this change?

Reply:
That's not a change...it's just reverting the check I was doing on the server side....